src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+
+    public boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if ((password == null | repeatPassword == null)
+                || (password.equals(repeatPassword) == false)
+                || (password.length() < 10 | repeatPassword.length() < 10)) {
```

Comment:
we can simplify this check:
- let's remove `repeatPassword == null`
- replase `password.equals(repeatPassword) == false` to `!password.equals(repeatPassword)`
- at the end should be checking for length (and only for `password` because if `!password.equals(repeatPassword)` return false, the sizes of `repeatPassword` and `password` will not be the same)

Reply:
fixed!